android/src/main/java/com/auth/react/AzureAuthModule.java

```diff
@@ -50,21 +50,10 @@ public String getName() {
 
     @ReactMethod
     public void showUrl(String url, boolean closeOnLoad, Callback callback) {
-        final Activity activity = getCurrentActivity();
-
-        this.callback = callback;
-        this.closeOnLoad = closeOnLoad;
-
```

Comment:
you have thrown away crucial lines to save the callback and closeOnLoad flag.

Reply:
Oh got you